README.md

```diff
@@ -43,6 +43,17 @@ This produces:
 {"level":30,"time":1531171082399,"msg":"hello child!","pid":657,"hostname":"Davids-MBP-3.fritz.box","a":"property","v":1}
 ```
 
+Also, the log level methods [return reasonable values](/docs/api.md#loggingmethodreturns):
```

Comment:
is "reasonable value" a computer science term? We can just say it returns the value of the `msg` key I think

Reply:
No, I chose the term intentionally.  It doesn't return the value of the `msg` key.  It returns values as described in the changes I made to the docs:  if `typeof` the first argument is `object`, it returns that object, else it returns the first string argument, unless there are multiple arguments, in which case it returns the interpolated string.  It's just that that's a mouthful and the docs describe it ok.  I'll add that description to the docs.

Reply:
I think "reasonable value" is an unorthodox term to use, and sends you on a mental pathway unnecessarily wondering if thats A Thing or just an unusual turn of phrase. Let's just describe what it is